src/proto/trainer.proto

```diff
@@ -49,6 +49,10 @@ message Trainer {
   // background.
   int64 num_parallel_readers = 3;
 
+
+  // If false, trainers will have their trainer rank mixed into their random seed.
+  bool random_init_trainers_identically = 4;
```

Comment:
Can we put this under the "Advanced options"?

Reply:
sure